tests/integration/parity/common.py

```diff
@@ -42,39 +42,10 @@ def test_eth_uninstallFilter(self, web3):
         pytest.xfail('eth_uninstallFilter calls to parity always return true')
         super().test_eth_uninstallFilter(web3)
 
-    def test_eth_replaceTransaction(self, web3, unlocked_account):
```

Comment:
I'm seeing that the inclusion of these tests produces the `FileNotFoundError: [Errno 2] No such file or directory: 'parity'`. Curious to get some more context around this though.

Reply:
Hmm, that shouldn't be the case. Are you seeing that locally? If so, it may be because you don't have parity installed?